docs/changelog.md

```diff
@@ -1,6 +1,20 @@
 # TBD
+
 ### Features
 * Port over the relevant bits of documentation from `kurtosis-libs`
+* Added new method `GetServiceInfo` in `api_container_service.proto` file
+* Added new method `GetServiceContext` in `NetworkContext` which returns relevant information about the service
+
+### Changes
+* Removed the services map from `NetworkContext` which converts it in stateless network representation
+
+### Breaking Changes
+* Removed `Service` interface
```

Comment:
So when we have breaking changes, we always want to give users remediation steps. In this case, I'd put "Removed `Service` interface; users should communicate with the service directly or use a custom client (e.g. ElasticsearchClient)"

Reply:
I got you